modules/osdk-cli-reference-generate-bundle.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-bundle_{context}"]
+= bundle
+
+The `generate bundle` subcommand generates a set of bundle manifests, metadata, and a `bundle.Dockerfile` file for your Operator project. Typically, the `generate kustomize manifests` subcommand would be run first to generate the input link:https://kustomize.io/[Kustomize] bases that are consumed by the `generate bundle` subcommand.
+
+.`generate bundle` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`--channels` (string)
+|Comma-separated list of channels to which the bundle belongs. The default is `alpha`.
+
+|`--crds-dir` (string)
+|Root directory for `CustomResoureDefinition` manifests
+
+|`--default-channel` (string)
+|The default channel for the bundle
+
+|`--deploy-dir` (string)
+|Root directory for Operator manifests, such as deployments and RBAC. This directory is different from the directory passed to the `--input-dir` flag.
+
+|`-h`, `--help`
+|Help for `generate bundle`
+
+|`--input-dir` (string)
+|Directory from which to read an existing bundle. This directory is the parent of your bundle `manifests` directory and is different from the `--deploy-dir` directory.
+
+|`--kustomize-dir` (string)
+|Directory containing Kustomize bases and a `kustomization.yaml` file for bundle manifests. The default is `config/manifests`.
+
+|`--manifests`
+|Generate bundle manifests
+
+|`--metadata`
+|Generate bundle metadata and Dockerfile
+
+|`--output-dir` (string)
+|Directory where to write the bundle
+
+|`--overwrite`
+|Overwrite the bundle metadata and Dockerfile if they exist. The default is `true`.
+
+|`--package` (string)
+|Package name for the bundle
+
+|`-q`, `--quiet`
+|Run in quiet mode
+
+|`--stdout`
+|Write bundle manifest to standard out
+
+|`--version` (string)
+|Semantic version of the Operator in the generated bundle. Set only when creating a new bundle or upgrading the Operator.
```

Comment:
```suggestion
|Semantic version of the Operator in the generated bundle. Set only when creating a bundle or upgrading the Operator.
```
Generally, "create ... new" is redundant, but if there's nuance in this use of new, please keep it.